README.md

```diff
@@ -39,6 +39,7 @@ Options:
   --dht_peer_maxAge, --dpa   Max age for peers in DHT                   [number]
   --cache_maxAge             Maximum cache age                          [number]
   --dnl, --dht_nodeLiveness  Interval in ms to check for dead nodes     [number]
+  --mps, --max_payload_size  Interval in ms to check for dead nodes     [number]
```

Comment:
`max_payload_size` -> `check_maxPayloadSize`
`mps` -> I would drop the short name